source/PlayerInfoPanel.cpp

```diff
@@ -700,21 +700,30 @@ void PlayerInfoPanel::DrawFleet(const Rectangle &bounds)
 
 	// Table attributes.
 	Table table;
-	for(const auto &col : columns)
-		table.AddColumn(col.offset, col.layout);
+	static const int GUTTER = 3;
+	int offset = 0;
+	for(const auto &column : columns)
+	{
+		table.AddColumn(offset + (column.layout.align == Alignment::RIGHT
+			? column.layout.width : 0), column.layout);
+		offset += column.layout.width + GUTTER;
+	}
 
 	table.SetUnderline(0, 730);
 	table.DrawAt(bounds.TopLeft() + Point(10., 8.));
 	table.DrawUnderline(dim);
 
 	// Header row.
 	const Point tablePoint = table.GetPoint();
+	const double rowHeight = table.GetRowSize().Y();
+	offset = 0;
 	for(const auto &column : columns)
 	{
-		Rectangle zone = Rectangle(
-			tablePoint + Point((column.offset + column.endX) / 2, table.GetRowSize().Y() / 2),
-			Point(column.layout.width, table.GetRowSize().Y())
+		Rectangle zone = Rectangle(tablePoint + Point(offset, 0) +
+			.5 * Point(column.layout.width - GUTTER, rowHeight),
```

Comment:
Why change line breaking?
```suggestion
		Rectangle zone = Rectangle(
			tablePoint + Point(offset, 0) + .5 * Point(column.layout.width - GUTTER, rowHeight),
```

Reply:
I don't recall doing so intentionally. It may have been Prettier, or perhaps I modified the line and later reversed it and didn't put it back exactly. Happy to blame Prettier though. :-)